src/en/proposals/errant-vox-species.md

```diff
@@ -0,0 +1,94 @@
+ 
+ - They have their own blood type, and when that system is expanded, it should be incompatible with "normal blood"
+ - Possibly their brain, (it actually being a piece of tech), is automatically ready to be placed into a borg chassis
+
+  - They are small/unassuming/fragile.
+    - Possibly a dwarf-sized sprite? (when you sprite it)
+    - They take ~ +25% damage from all Bruise
+    - They get drunk more easily
+    - Maybe they are worse at pulling things?
+    - They are easier to shove
+    - I thought they would deal sub-par damage but there have been some fairly strong feelings on Discord that their claws should be vicious claws. Maybe an exception can be made here, if they take more brute damage they won't really have an advantage in a brawl anyway. Unarmed damage probably isn't really all that important in a real fight anyway)
+  
+- They should have some actual mechanic involving screams (any such ability should consider that the hotbar cooldown does not apply to emoted screams. They might need to be an actual ability that also does a scream, rather than apply on any scream)
+  - they could have the ability to discern the positions of screams within a range, even through walls
+  - they could discern some information from screams, such as the coarse Health state of the person? (Good, Okay, Poor, Bad, Danger)
+  - their screams could apply a ! popup on nearby people with no additional effect?
+
+- They have a weird dietary range.
+  - It has been suggested that they be able to butcher without any tools. Very messily.
+  - Their thirst can only be satiated with welding fuel?
+  - Water makes them drunk
+  - Maybe "normal food" is less satiating to them than the weird stuff (if that can be categorized in some way)
+  
+- This was just an idea I toyed around with but they could have a mechanic based on hugs. However we want to explain it, from a deep psychological need based on their pack hunter ancestry, to them actually sapping small amounts of bioelectrical energy from others with surface contact (and reducing the other person's life expectancy by 1 to 3 seconds. Don't worry you'll be fiiiine it hardly matters.). The point is that they can't hug themselves and this will make them either hug every single crewmember they see (in hopes of reciprocation) and become known as having no concept of personal space, or threaten you with a fireaxe for hugs. Both of these outcomes are weird which is good.
+  - They get a sad vox face debuff on the alert HUD after going X minutes without getting a hug. What a terrible fate 
+  - Their bruise regeneration (that all organics have at low damage levels) could stop?
+  - Maybe their hunger meter depletes faster when debuffed
+
+# When Vox?
+I have every intention to work on the actual implementation of the species and will create a roadmap for getting to roundstart vox once the design direction is settled
```

Comment:
That's great to hear! Next steps imo is making another pass on your proposal and hammering down how YOU think vox should work. You've laid out alot of great concepts/ideas but you haven't really commited to any of them fully or developed them out yet. For this to be an actionable design document you're going to need to decide on what exactly you want for Vox and present it in a concrete way. 

Otherwise, this is a great first pass and I really like some of the ideas you proposed! I look forward to seeing your V2 of the doc and reading through your interpretation of how vox should be implemented :)

Reply:
First, thank you for the input! That's exactly what I'm here for. You are entirely correct that this is not directly actionable, as I was not writing it to that standard. Because, and I think this is important to point out: this was actually not listed as a requirement for a design submission.  ((I assumed that it's enough to settle on broad strokes here, and I intended to settle on the specifics on the PR to add them)
"You should be able to hand it to a coder and they should be able to implement it" is, to be honest, an extremely sensible requirement. But the first time I heard about that, that it should be this specific, was like a week after my proposal was already submitted. I heard it in a random discord discussion, and now I have heard it here. I do not say this because I feel a need to be deffensive about it, but rather because I think that this should be clarified on the [proposal template](https://docs.spacestation14.com/en/templates/proposal.html). I don't know if this was just assumed to be obvious to everyone (I am now evidence that it's not), or that page is just old (it predates the design group by far, of course). But making this more clear would probably be helpful for future contributors.

I will mull over the specific details and try to nail them down more concretely.